G1ANT.Addon.Selenium/G1ANT.Addon.Selenium.csproj

```diff
@@ -36,8 +36,8 @@
     <Reference Include="BouncyCastle.Crypto, Version=1.8.4.0, Culture=neutral, PublicKeyToken=0e99375e54769942">
       <HintPath>..\packages\BouncyCastle.1.8.4\lib\BouncyCastle.Crypto.dll</HintPath>
     </Reference>
-    <Reference Include="G1ANT.Language, Version=4.101.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\G1ANT.Language.4.101.0\lib\net452\G1ANT.Language.dll</HintPath>
+    <Reference Include="G1ANT.Language, Version=4.105.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\G1ANT.Language.4.105.0-alpha0065\lib\net452\G1ANT.Language.dll</HintPath>
```

Comment:
we need to reference to the public NuGet G1ANT.Language package, so wait for the official version 4.106.0 with DataTableStructure support

Reply:
We've discussed this issue with @pwitusik and for now we can use this reference and wait for a publish. There is another task to update this reference after publish. DataTableStructure is not yet in production that is why I'm using this one, just for the debugging purposes.